src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -24,6 +24,7 @@
     <Compile Include="$(SharedDir)StringComparers.cs" Link="Utils\StringComparers.cs" />
     <Compile Include="$(SharedDir)KnownFormats.cs" Link="Utils\KnownFormats.cs" />
     <Compile Include="$(SharedDir)CircularBuffer.cs" Link="CircularBuffer.cs" />
+    <Compile Include="$(SharedDir)ContainerImageTags.cs" Link="Utils\ContainerImageTags.cs" />
```

Comment:
I don't think this file needs to be in the `Shared` folder. I think you can just put it top level in Aspire.Hosting, and link to there from the tests.